src/main/groovy/ch/ubique/gradle/ubdiag/BuildPlugin.groovy

```diff
@@ -52,7 +51,9 @@ class BuildPlugin implements Plugin<Project> {
 			// Add generated icon path to res-SourceSet. This must be here otherwise it is too late!
 			AndroidSourceSet sourceSet = android.sourceSets.maybeCreate(flavor.name)
 			sourceSet.res { AndroidSourceDirectorySet res ->
-				res.srcDir("${project.buildDir}/generated/res/icon/${flavor.name}/")
+				android.buildTypes.all { buildType ->
+					res.srcDir("${project.buildDir}/generated/res/icon/${flavor.name}/${buildType.name}/")
+				}
```

Comment:
Funktioniert das auch wenn man einen zusätzlichen build type definiert? Meines Wissens gibt `buildTypes.all` nur die zurück die genau zu diesem Zeitpunkt definiert sind.

Reply:
Hab es mal getestet mit einem zusätzlichen BuildType. Wird auch erkannt:

> Configure project :app
BuildType -> debug
BuildType -> release
BuildType -> debug
BuildType -> release
BuildType -> bliblablub
BuildType -> bliblablub

> Task :prepareKotlinBuildScriptModel UP-TO-DATE

BUILD SUCCESSFUL in 8s